subsys/bluetooth/controller/Kconfig

```diff
@@ -309,6 +312,21 @@ config BT_CTLR_TX_PWR_MINUS_40
 
 endchoice
 
+config BT_CTLR_TX_PWR_SET
+	int "Set TX power (dBm)"
+	range -127 127
+	default 0
+	depends on BT_CTLR_TX_PWR_0
+	help
+	  Request a TX power in dBm. The power level set will be equal to or
+	  less than the one requested, based on the values supported by the
+	  hardware used. Values outside the supported range will be set to the
+	  nearest supported value.
+	  The value set here represents the actual power level fed to the
+	  antenna. When a Front-End Module is used, gain values for the SoC and
```

Comment:
`When a Front-End Module is used` does it mean, if FEM is not used, that gain isn't calculated automatically?

Reply:
If a FEM is not used there should be no gain, the tx powers supported by the SoC will be used.

Reply:
Ok,  so now it depends how are we going to use it in future. Referring to @rugeGerritsen explanation, this may be used instead of the currently used option.  Then this option would represent TX output power that is combined of Radio TX PWR + FEM gain. If FEM is not available it would represent the same value as 'BT_CTLR_TX_PWR'. If that was my decision,  I would go with this approach.

Also there is no dependency on FEM availability. I assume this is implicit for end user, hece it is another reason to represent total TX power or,  if no FEM,  the same value as 'BT_CTLR_TX_PWR'.

Reply:
Yes, but to clarify. You can also use a FEM with `BT_CTLR_TX_PWR` in the same way. You are just very limited on what you can select, so it is not very useful. `BT_CTLR_TX_PWR_SET` has no special behavior beyond letting the user set any tx power.